kod/object/passive/ornobj.kod

```diff
@@ -735,14 +749,143 @@ messages:
    {
       local f;
 
-      f = LOOK_NO;
+      f = 0;
+      if NOT pbLook { f = f | LOOK_NO; }
       if pbHanging { f = f | OF_HANGING; }
       if pbFlicker { f = f | FLICKERING_YES; }
       if pbNoMoveOn { f = f | MOVEON_NO; }
       
       return f;
    }
 
+   ShowDesc()
+   {
+      if psCustomDesc <> $
+      {
+         AddPacket(4, oo_custom_template_rsc, 4, psCustomDesc);
+
+         return;
+      }
+
+      propagate;
+   }
+
+   StartDecayTimer(d=0, h=0, m=0, s=0)
+   "Admin supported\n"
+   "Starts a timer for this object which will automatically delete it upon expiring.\n"
+   "Can specify the time using h for hours, m for minutes, and/or s for seconds."
+   {
+      local iTime;
+
+      iTime = d * 86400000 + h * 3600000 + m * 60000 + s * 1000;
+
+      if (iTime > 0)
+      {
+         ptDecayTimer = CreateTimer(self, @DecayTimerExpired, iTime);
+      }
+      else
+      {
+         Send(self, @ClearDecayTimer);
```

Comment:
This is a weird behavior.. pass in a negative value and it clears the existing timer?  I think instead, negative values should give an error.

Reply:
I agree, not sure why I even did it that way in the first place.  Changed to a debug message.